program/src/processor/distribute_rewards.rs

```diff
@@ -68,11 +69,10 @@ pub fn process_distribute_rewards(
 
     if rewards_per_token != 0 {
         // updates the accumulated stake rewards per token
-        let accumulated = config
-            .accumulated_stake_rewards_per_token()
+        let accumulated = <PodU128 as Into<u128>>::into(config.accumulated_stake_rewards_per_token)
```

Comment:
nit: I typically find it easier to read `from` instead of into in a situation like this, ie
```suggestion
        let accumulated = u128::from(config.accumulated_stake_rewards_per_token)
```

Reply:
Nice, definitely better!